target/linux/ramips/image/mt76x8.mk

```diff
@@ -159,6 +159,15 @@ define Device/cudy_wr1000
 endef
 TARGET_DEVICES += cudy_wr1000
 
+define Device/cudy_tr1200
+  IMAGE_SIZE := 15872k
+  DEVICE_VENDOR := Cudy
+  DEVICE_MODEL := TR1200
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci kmod-usb-ledtrig-usbport kmod-mt7615e kmod-mt7663-firmware-ap
```

Comment:
Please split them into two lines. It's too long.

Reply:
done